instructors/Template.ipynb

```diff
@@ -0,0 +1,248 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<a name=\"top\"></a>\n",
+    "<div style=\"width:1000 px\">\n",
+    "\n",
+    "<div style=\"float:right; width:98 px; height:98px;\">\n",
+    "<img src=\"https://raw.githubusercontent.com/Unidata/MetPy/master/src/metpy/plots/_static/unidata_150x150.png\" alt=\"Unidata Logo\" style=\"height: 98px;\">\n",
+    "</div>\n",
+    "\n",
+    "<h1>YOUR NOTEBOOK TITLE</h1>\n",
+    "<h3>Unidata AMS 2021 Student Conference</h3>\n",
+    "\n",
+    "<div style=\"clear:both\"></div>\n",
+    "</div>\n",
+    "\n",
+    "<hr style=\"height:2px;\">\n",
+    "\n",
+    "Along with your notebook, be sure to commit a small example output image from your code and link to it HERE (in the source), or link to an external image (logo, etc.) via URL. Remove this line after!\n",
+    "<div style=\"float:right; width:250 px\"><img src=\"../../instructors/images/Template_preview.png\" style=\"height: 300px;\"></div>\n",
```

Comment:
Looks like this goes up two directories before coming back down into the `instructors/` directory, which causes the image to be missing in [github preview](https://github.com/Unidata/pyaos-ams-2021/blob/19cd1fc599d80c7f9e2134f5412f00878d40d2d8/instructors/Template.ipynb), which is ok depending on where the template is to be copied. This will work if we are linking from `notebooks/<subcategory>/`, right? If so, is it worth adding another nested level for the template to live (say, `instructors/training-template/Template.ipynb`) so that the preview image works when viewing the template through github as well as from the location where it is likely to get copied to?

Reply:
Alright, done and also included some info about `alt` text I had removed as it may be helpful for certain users.